ffi/tests/getset.rs

```diff
@@ -3,15 +3,17 @@
 use std::mem::MaybeUninit;
 
 use getset::{Getters, MutGetters, Setters};
-use iroha_ffi::{ffi_export, FfiType};
 
-#[derive(Debug, Clone, PartialEq, Eq, FfiType)]
+#[derive(iroha_ffi::FfiType)]
```

Comment:
the situation is such that `FfiType` needs to be declared as the outermost macro so it observes other derives. This is so that if getset derives are observed it will automatically produce FFI functions